### DISCUSSION TITLE
sfdx-git-delta with locally installed @salesforce/cli

### DISCUSSION THREAD

**Discussion by anthogiuliano** [scolladon/sfdx-git-delta#939] [Q&A] [answered]
I'm interested in using `@salesforce/cli` as a local npm package and calling it using `npx`, instead of installing `@salesforce/cli` as a global package. But it seems like `sfdx-git-delta` expects the cli to be installed globally and doesn't work without that. Is there any way to control where it looks for the salesforce cli?

**Comment by scolladon** [comment, marked as answer]
Hi @anthogiuliano !

When building a plugin we do not specify where the plugin is going to be installed.
So this is not up to me to tell my plugin to expect where the cli should be installed.

I think the question (_How can I install the cli and related plugins locally to a package_) should be raised at `@salesforce/cli` directly.
Feel free to link the question to this discussion ! I'm curious about how to achieve this.

**Reply by anthogiuliano** [reply]
That makes sense, thank you!
